ecosystem/sep-0041.md

```diff
@@ -200,6 +202,12 @@ The event has topics:
 The event has data:
 
 - `i128` the amount transferred.
+- or `map` containing entries with `Symbol` keys, and values defined as
+  - `amount: i128` the amount minted.
+  - `to_muxed_id: Option<u64 | String | BytesN<32>>` void, or entry absent, if
+    no muxed ID, or the u64 muxed ID, or for historical legacy reasons, the
+    string or bytes memo for Stellar assets.
+  - Other entries allowed as defined by the implementation.
```

Comment:
Is it necessary to call out that other keys may be present in the map in this SEP? As a general rule I think its implied that other key-val pairs can be included in maps referenced in SEPs, but they aren't a part of the specification and therefore don't need to be explicitly acknowledged.